frontend/src/layout/navigation/TopBar/SitePopover.tsx

```diff
@@ -55,14 +58,17 @@ function AccountInfo(): JSX.Element {
                     {user?.email}
                 </div>
             </div>
-            <Link
-                to={urls.mySettings()}
-                onClick={closeSitePopover}
-                className="SitePopover__side-link"
-                data-attr="top-menu-item-me"
-            >
-                Manage account
-            </Link>
+            <Tooltip title="Account settings">
```

Comment:
This tooltip should be placed on the left:
<img width="164" alt="Screen Shot 2022-03-02 at 15 55 04" src="https://user-images.githubusercontent.com/4550621/156386408-50ea9e2e-f144-43f0-ba24-69e343aba7db.png">